TCAT/Views/WalkWithDistanceIcon.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  WalkWithDistanceIcon.swift
+//  TCAT
+//
+//  Created by Monica Ong on 3/7/18.
+//  Copyright © 2018 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class WalkWithDistanceIcon: UIView {
+
+    // MARK: View vars
+    
+    var walkIcon: UIImageView
+    var travelDistanceLabel: UILabel
+    
+    // MARK: Spacing vars
+    
+    let walkIconAndDistanceLabelVerticalSpace: CGFloat = 2.0
+
+    init(walkIcon: UIImageView, travelDistanceLabel: UILabel) {
```

Comment:
does this need to be a parameter, does it change? If so, maybe an enum or color or whatever distinguishes it